### DISCUSSION TITLE
antd-v5, FormDialog 要怎么改变 footer button 的主题颜色？

### DISCUSSION THREAD

**Discussion by JE-lee** [alibaba/formily#3959] [Q&A 2.x]
<img width="1291" alt="image" src="https://github.com/alibaba/formily/assets/19794813/addc1a32-393e-45bc-8c06-f17ecd7e877f">
如上图，使用 FormDialog 打开的表单，默认的 footer 按钮的颜色好想不跟随顶层主题的设置。

**Comment by yanhao98** [comment]
要用一下。
FormDialog.Portal
